src/main/java/de/tum/in/www1/artemis/web/rest/ExampleSubmissionResource.java

```diff
@@ -145,4 +151,25 @@ public ExampleSubmissionResource(ExampleSubmissionService exampleSubmissionServi
 
         return ResponseEntity.ok().headers(HeaderUtil.createEntityDeletionAlert(applicationName, true, ENTITY_NAME, id.toString())).build();
     }
+
+    /**
+     * POST exercises/:exerciseId/example-submissions/import : Import exampleSubmission.
+     *
+     * @param exerciseId        the id of the corresponding exercise
+     * @param submissionId        the submission id to be imported as an example submission
+     * @return the ResponseEntity with status 200 (OK) and the Result as its body, or with status 4xx if the request is invalid
+     */
+    @PostMapping("exercises/{exerciseId}/example-submissions/import")
+    @PreAuthorize("hasRole('INSTRUCTOR')")
+    public ResponseEntity<ExampleSubmission> importExampleSubmission(@PathVariable Long exerciseId, @RequestBody Long submissionId) {
+        log.debug("REST request to import ExampleSubmission : {}", submissionId);
+
+        Exercise exercise = exerciseRepository.findByIdElseThrow(exerciseId);
+
+        authCheckService.checkHasAtLeastRoleForExerciseElseThrow(Role.INSTRUCTOR, exercise, null);
+
```

Comment:
that edge case seems unlikely, but yeah can be checked

Reply:
While it seems unlikely it is a security issue because, in theory, one could import that way a student submission from another course even into this exercise which would allow this person to look at submissions, potentially even exam submissions, that there person doesn't have access to.

Reply:
hmm, cool. I'm convinced. 😄 need to check submission repo what I can do about that. cause now I have only submissionId and exerciseId,

Reply:
I think you can check whether `Submission.of(submissionId).participation.exercise.id == exerciseId`.
But I might be too premature on that.